api/templates/viewer.html

```diff
@@ -134,6 +134,7 @@
             },
         ];
         var lava_urls = {
+            'lava-baylibre': 'lava.baylibre.com',
```

Comment:
```suggestion
            "lava-baylibre": "lava.baylibre.com",
```

For consistency ;)

Reply:
Done, thanks!

Reply:
Did you forget to commit and/or push? I don't see any change here

Reply:
Editor glitch, fixed now